Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -99,6 +101,19 @@ private void configureMotorTeam(
         master.createTelemetryProperties(masterName);
         
         this.updateMotorPidValues(master);
+        
+        // Master Config
+        master.setNeutralMode(NeutralMode.Coast);
```

Comment:
Hmmm... does this need to be done for the follower as well?

Reply:
Yeah it'd be safe to add that in